org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/repositories/AbapGitView.java

```diff
@@ -167,7 +185,32 @@ public Image getImage(Object obj) {
 	 */
 	@Override
 	public void createPartControl(Composite parent) {
-		setupViewer(parent);
+		final Composite composite = new Composite(parent, SWT.NONE);
+		GridData gd = new GridData(SWT.FILL, SWT.FILL, true, true);
+		composite.setLayoutData(gd);
+		GridLayout layout = new GridLayout(1, false);
+		composite.setLayout(layout);
+
+		//Create Filter Text Box
+		this.searchText = new Text(composite, SWT.SEARCH | SWT.CANCEL);
+		this.searchText.setMessage(Messages.AbapGitView_Type_Filter_Text);
+		this.searchText.setLayoutData(new GridData(GridData.GRAB_HORIZONTAL | GridData.HORIZONTAL_ALIGN_FILL));
```

Comment:
[SUGGESTION] We can move this code to a separate method `setUpFilterBox()`, and can even move the searchText listener to that method.

Reply:
moved the code to `setUpFilterBox()` method.